ocaml/xcp-rrdd/bin/rrdp-scripts/rrdd-plugins-init

```diff
@@ -37,9 +34,8 @@ start() {
 		
 		if [ -e ${SUBSYS_FILE} ]; then
 			if [ -e ${PID_FILE} ] && [ -e /proc/`cat ${PID_FILE}` ]; then
-				echo -n $"cannot start ${NAME}: already running."
-				failure $"cannot start ${NAME}: already running."
-				echo
+				echo $"cannot start ${NAME}: already running."
```

Comment:
I see that on the shellcheck commit, dollar signs in the echo commands have been removed. What is their function here and why have they been removed elsewhere but not here?

Reply:
Found it:
```
 A double-quoted string preceded by a dollar sign ($) will cause the string to be translated according to the current locale.  If the current
       locale is C or POSIX, the dollar sign is ignored.  If the string is translated and replaced, the replacement is double-quoted.
```
I don't know where these ssupposed translation exist. Is xenserver configured to several languages to support this feature?